packages/vendure-plugin-invoices/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.1.4 (2024-08-01)
+
+- Show `Regenerate invoice` button with warning when order total is greater than the total of the latest invoice ( see #485)
```

Comment:
This should be "When order total differs from the latest invoice total" or something similar. It can also be the case that an item was removed from the Order